### ISSUE TITLE
`alevin-fry view` panicked at the prospect of providing help (or working)

### ISSUE THREAD

**Issue by j-andrews7** [COMBINE-lab/alevin-fry#48]
I am trying to determine what all can be done to/with the RAD file output via a typical `alevin-fry quant` run, but I am struggling to find much info about the `view` command. Ultimately, I was hoping to convert back to BAM to spotcheck induction of a mutation in our model system.

I was hoping the command line help would yield some insight, but it throws an error:

```
alevin-fry view -h
thread 'main' panicked at '`help`s `-h` conflicts with `header`.

To change `help`s short, call `cmd.arg(Arg::new("help")...)`.', /opt/conda/conda-bld/alevin-fry_1645548081272/work/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-3.1.1/src/build/command.rs:4249:25
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

I looked through the repo code to find the proper args (`-r`, etc), but it still tosses the same error and doesn't seem this subcommand is functional yet.

I'm guessing you're well aware of this, and it's uncommon for people to want to interact with the RAD files directly. But it may be a good idea to remove it from the CLI help subcommand list until it's usable.

**Comment by rob-p** [comment]
Hi @j-andrews7,

  Thanks for the detailed report. Indeed, while this is not currently part of any standard workflow, there should certainly be a functional help message and there should be no panic!  We'll try to clean this up for the next release.

**Comment by DongzeHE** [comment]
Hi @rob-p, yes I will do this today. Should I directly create a PR to the master branch when I am done?

**Comment by rob-p** [comment]
This is now fixed in v0.5.1 — thanks again for the report!
